auth/auth-server/src/main.rs

```diff
@@ -163,8 +174,22 @@ async fn main() {
     .unwrap()
     .expect("Failed to setup token remaps");
 
+    // Build an Arbitrum client
+    let wallet =
+        LocalWallet::from_str(DUMMY_PRIVATE_KEY).expect("Failed to create wallet from private key");
+    let arbitrum_client = ArbitrumClient::new(ArbitrumClientConfig {
+        darkpool_addr: "0x9af58f1ff20ab22e819e40b57ffd784d115a9ef5".to_string(),
+        chain: args.chain_id,
+        rpc_url: "https://arb-sepolia.g.alchemy.com/v2/nnNuwdTdUm-M2xoUq11wq7brieciW-98"
+            .to_string(),
+        arb_priv_keys: vec![wallet],
+        block_polling_interval_ms: 100,
+    })
+    .await
+    .unwrap();
+
     // Create the server
-    let server = Server::new(args).await.expect("Failed to create server");
+    let server = Server::new(args, arbitrum_client).await.expect("Failed to create server");
```

Comment:
Moving forward, perhaps a `TelemetryClient` entity would be a better place to instantiate the Arbitrum client since we'll be adding more complex logic regarding quote comparison and on-chain queries